tests/benchdnn/inputs/conv/test_conv_gpu_ci

```diff
@@ -1,83 +1,68 @@
 --reset
 
---mb=2,16
 --stag=any,axb
---dtag=any,axb
---alg=direct
 
 # Training
 ## Forward
 --dir=FWD_B
 --dt=f32,bf16,f16
---attr-post-ops=,relu,sum:0.25+relu:0.5
---batch=shapes_basic_gpu
+--attr-post-ops=,sum:0.25+relu:0.5
+--batch=shapes_ci_gpu
+--attr-post-ops=
+
 ## Backward
 --dir=BWD_D,BWD_WB
 --dt=f32,bf16
---attr-post-ops=
---batch=shapes_basic_gpu
+--batch=shapes_ci_gpu
 
-## Fpmath-mode
+## FP math mode
 --dir=FWD_B,BWD_D,BWD_WB
 --dt=f32
 --attr-fpmath=tf32,bf16
---attr-post-ops=
---batch=shapes_basic_gpu
+--batch=shapes_ci_gpu
+--attr-fpmath=
 
-# Inference
---reset
+# int8
+## Type combinations
 --dir=FWD_I
---attr-scales=,src:common:0.25,wei:per_oc,dst:common:2,src:common:0.25+wei:per_oc
-
-## All inference configs
 --dt=s8:s8:f32,s8:s8:bf16,s8:s8:f16,s8:s8:s32,s8:s8:s8,s8:s8:u8, \
       u8:s8:f32,u8:s8:bf16,u8:s8:f16,u8:s8:s32,u8:s8:s8,u8:s8:u8
---batch=shapes_basic_gpu
+--batch=shapes_ci_gpu
 
 ## Attributes
---dt=u8:s8:s32,s8:s8:f16
---stag=any,axb
---dtag=any,axb
---attr-post-ops=,sum:0.5:3+add:f32:per_oc+add:f32:per_tensor
---attr-zero-points=
---batch=shapes_basic_gpu
---attr-post-ops=
+--dir=FWD_I
+--dt=u8:s8:s32
+--attr-scales=,src:common:0.25,wei:per_oc,src:common:0.25+wei:per_oc+dst:common:2
+--attr-post-ops=,sum:0.5:3+add:f32:per_oc+mul:f16:per_tensor
 --attr-zero-points=,src:common:2+dst:common:1,src:per_dim_1+dst:per_dim_1
---batch=shapes_basic_gpu
+--batch=shapes_ci_gpu
+--attr-scales=
+--attr-post-ops=
+--attr-zero-points=
 
-# Wino
---reset
---mb=2,32
---stag=any,axb
+# Winograd
 --alg=wino
```

Comment:
Given that the Winograd implementation is not being actively developed, it is likely reasonable to just drop it from CI testing and rely on nightly for its validation.

Reply:
Removed.